### ISSUE TITLE
Rounding tracks hangs and never finishes

### ISSUE THREAD

**Issue by qubist** [mitxela/kicad-round-tracks#7]
```
Application: KiCad PCB Editor
Version: (6.99.0-1716-g37a2ee7d03), release build
Libraries:
    wxWidgets 3.1.5
    FreeType 2.12.0
    HarfBuzz 4.2.0
    FontConfig 2.14.0
    libcurl/7.77.0 SecureTransport (LibreSSL/2.8.3) zlib/1.2.11 nghttp2/1.42.0
Platform: macOS Version 12.1 (Build 21C52), 64 bit, Little endian, wxMac
Build Info:
    Date: Apr 19 2022 22:10:12
    wxWidgets: 3.1.5 (wchar_t,wx containers)
    Boost: 1.78.0
    OCC: 7.6.1
    Curl: 7.64.1
    ngspice: 35
    Compiler: Clang 12.0.0 with C++ ABI 1002
Build settings:
    KICAD_USE_OCC=ON
    KICAD_SPICE=ON
```

Error shown in "Traceback of Python Script Errors" window:

```
**********************************
  File "/Applications/KiCad/KiCad.app/Contents/Frameworks/Python.framework/Versions/Current/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 779, in exec_module
  File "<frozen importlib._bootstrap_external>", line 916, in get_code
  File "<frozen importlib._bootstrap_external>", line 846, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Applications/KiCad/KiCad.app/Contents/SharedSupport/scripting/plugins/FPC_wizard.py", line 94
    pad_s0_pos = pcbnew.VECTOR2I( xpos, (int)shl_from_top )
                                             ^
SyntaxError: invalid syntax
```

**Comment by mitxela** [comment]
Sorry, I have not tested with 6.99. The python API is unstable in kicad nightly, and there are plans to completely replace it before the next major release, so I'm not sure I can commit the time to maintain a 6.99 branch.

If the crash also happens in KiCad 6, please let me know, and if possible share the design files (or just the part which causes the crash, if you can isolate it).

**Comment by qubist** [comment]
Thanks for the reply! I was having an issue with python scripting not working at all in 6.0.4 so I was trying on 6.99. Figured that version just didn't support the plugin. I redownloaded 6.0.4 and now all plugins (including this one) are working! The curved tracks look so great, thank you very much for developing this :)

<img width="464" alt="Screen Shot 2022-04-21 at 1 11 42 PM" src="https://user-images.githubusercontent.com/3607279/164514578-d673b73d-8f76-4490-85af-d2e7a768a6c6.png">
